src/utils/common-utils.js

```diff
@@ -60,6 +60,26 @@ var CommonUtils = { // jshint ignore:line
 
     cropDomain: function (domain) {
         return domain.replace("www.", "").replace(/:\d+/, '');
+    },
+
+    /**
+     * Force clear page cache
+     */
+    reloadPageBypassCache: function() {
+        var xhr = new XMLHttpRequest();
+        xhr.open('GET', window.location.href, true);
+
+        xhr.setRequestHeader('Pragma', 'no-cache');
+        xhr.setRequestHeader('Expires', -1);
+        xhr.setRequestHeader('Cache-Control', 'no-cache');
+
+        xhr.onreadystatechange = function() {
+            if (xhr.status === 200 && xhr.readyState === 4) {
```

Comment:
It will never reload if the status code is not 200, and it is possible in theory